### DISCUSSION TITLE
[✨] ServerRequestEvent needs httpVersion entry to solve HTTP/2 specific problems.

### DISCUSSION THREAD

**Discussion by genki** [QwikDev/qwik-evolution#172] [Proposals For Qwik]
### Is your feature request related to a problem?

When using dev-server as https, currently it warns like this:

```
(node:79236) UnsupportedWarning: Status message is not supported by HTTP/2 (RFC7540 8.1.2.4)
(Use `node --trace-warnings ...` to show where the warning was created)
```

This is happening because the server response body contains status message that is not allowed for HTTP/2 protocol.
So then it is necessary to provide the body conditional to the protocol version, but there is no informations about it in the ServerRequestEvent.

### Describe the solution you'd like

My suggestion is add `httpVersion: {major:number, minor:number}` to the `ServerRequestEvent` interface.
That can be acquired from `IncomingMessage`.

### Describe alternatives you've considered

Sorry, I couldn't come up with other solution.

**Comment by genki** [comment]
I have found this issue is still happening.
So I traced warning, the reason of the problem is found in the finalhandler.
I have sent a PR to fix it. 
https://github.com/pillarjs/finalhandler/pull/45

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.
